cultcargo/genesis/wsclean/wsclean-base.yml

```diff
@@ -159,15 +167,16 @@ lib:
           dtype: float
         maxuvw-m:
           dtype: float
+        temp-dir:
+          dtype: Directory
+          skip_freshness_checks: true
+          mkdir: false
```

Comment:
The reason I made this an output was because wsclean will not create directories i.e. runs will break on wsclean steps if `temp-dir` doesn't exist. Stimela does not honour `mkdir` on inputs.

Reply:
Fair enough. I'll move it back to outputs